exercise/templates/exercise/exercise.html

```diff
@@ -58,6 +58,8 @@
 
 {% include "exercise/_exercise_wait.html" %}
 
+{% include "exercise/_duplicate_submission_modal.html" %}
```

Comment:
Bootstrap docs recommend that the modal HTML is not nested inside other HTML elements (except the `<body>)`. Nesting causes some risk that inherited CSS styles would affect the modal in unintended ways. At least, you could move the modal include to the bottom of this file outside the `<div>` elements.

Reply:
I don't think this include is inside any other HTML element, so I left it there.